checks/raw/contributors.go

```diff
@@ -23,7 +23,8 @@ import (
 )
 
 // Contributors retrieves the raw data for the Contributors check.
-func Contributors(c clients.RepoClient) (checker.ContributorsData, error) {
+func Contributors(cr *checker.CheckRequest) (checker.ContributorsData, error) {
+	c := cr.RepoClient
```

Comment:
I think it's nice for consistency across the code. Wdut?

Reply:
There's already inconsistencies between repoclient and check request, in both the converted and unconverted checks.

Reply:
@laurentsimon Kind ping.

Reply:
I'm fine either way. I like the consistency if we decide to use vs the other across the code. Because there's inconsistency now does not mean we should not try to reduce it. But that's just my opinion. Not a blocker on my side.